src/app/core/shared/form-patterns.constants.ts

```diff
@@ -0,0 +1,2 @@
+//Patterns for form validations
+export const VALID_EMAIL_PATTERN = "^[a-zA-Z0-9.!#$%&\\' * +\\/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$";
```

Comment:
This doesn't appear to be the same pattern we were using previously in the `register-email-form.component.ts`.  Is there a reason you've changed it?

I think you need to update this pattern to be:
```
"^[a-zA-Z0-9.!#$%&\'*+\/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$"
```

That value is copied directly from the `register-email-form.component.ts` and is the same as the HTML5 spec recommended JavaScript pattern at https://html.spec.whatwg.org/multipage/input.html#valid-e-mail-address